### ISSUE TITLE
new panics on 0.31 with workers

### ISSUE THREAD

**Issue by Fenzland** [denoland/deno#3794]
### setTimeout in worker panicks
```javascript
// index.js
const worker= new Worker( './worker.js', { name:'foo', type:'module', }, );

worker.onmessage= ()=> {
	console.log( 'done', );
};

worker.postMessage( {}, );
```

```javascript
// worker.js
globalThis.onmessage= async ()=> {
	
	setTimeout( async ()=> {}, ); // this line cause the panic
	
	globalThis.postMessage( {}, );
	
	if( globalThis.Deno )
		return globalThis.workerClose();
	
	globalThis.close();
};
```
It runs fine on and before 0.30, but panick in 0.31:
```bash
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: TryFromTypeError { expected_type: "Uint32" }', src/libcore/result.rs:1165:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
fatal runtime error: failed to initiate panic, error 5
```

### open worker in worker panicks 
```javascript
// index.js
const worker= new Worker( './worker.js', { name:'foo', type:'module', }, );

worker.onmessage= ( { data:{ level, }, }, )=> {
	console.log( `done level ${level}`, );
};

worker.postMessage( { level:1, }, );
```

```javascript
// worker.js
globalThis.onmessage= async ( { data:{ level, }, }, )=> {
	
	console.log( `into level ${level}`, )
	
	if( level )
	{
		const worker= new Worker( './worker.js', { name:'foo', type:'module', }, );
		
		worker.onmessage= ( { data:{ level, }, }, )=> {
			console.log( `done level ${level}`, );
		};
		
		worker.postMessage( { level:level - 1, }, );
	}
	
	globalThis.postMessage( { level, }, );
	
	if( globalThis.Deno )
		return globalThis.workerClose();
	
	globalThis.close();
};
```
It runs fine on and before 0.30, but panick in 0.31:
```bash
into level 1
into level 0
done level 0
done level 1
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:378:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

#### Env
Ubuntu Linux 18.04
deno 0.31.0
v8 8.1.108
typescript 3.7.2

**Comment by kevinkassimo** [comment]
cc @bartlomieju since there are major refactoring of worker recently

**Comment by kitsonk** [comment]
It would be good to get a full backtrace too, `RUST_BACKTRACE=1 deno run index.js` for example.

The first panic looks like a missing op ID.

**Comment by isAdrisal** [comment]
Confirm same panic/regression on v0.31.0 using the first setTimeout example.

RUST_BACKTRACE output below.

Windows 10 1909
deno 0.31.0
v8 8.1.108
typescript 3.7.2

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: TryFromTypeError { expected_type: "Uint32" }', src\libcore\result.rs:1165:5
stack backtrace:
   0: CrashForExceptionInNonABICompliantCodeRange
   1: CrashForExceptionInNonABICompliantCodeRange
   2: CrashForExceptionInNonABICompliantCodeRange
   3: CrashForExceptionInNonABICompliantCodeRange
   4: CrashForExceptionInNonABICompliantCodeRange
   5: CrashForExceptionInNonABICompliantCodeRange
   6: CrashForExceptionInNonABICompliantCodeRange
   7: CrashForExceptionInNonABICompliantCodeRange
   8: CrashForExceptionInNonABICompliantCodeRange
   9: CrashForExceptionInNonABICompliantCodeRange
  10: <unknown>
  11: <unknown>
  12: CrashForExceptionInNonABICompliantCodeRange
  13: CrashForExceptionInNonABICompliantCodeRange
  14: CrashForExceptionInNonABICompliantCodeRange
  15: CrashForExceptionInNonABICompliantCodeRange
  16: CrashForExceptionInNonABICompliantCodeRange
  17: CrashForExceptionInNonABICompliantCodeRange
  18: CrashForExceptionInNonABICompliantCodeRange
  19: CrashForExceptionInNonABICompliantCodeRange
  20: CrashForExceptionInNonABICompliantCodeRange
  21: CrashForExceptionInNonABICompliantCodeRange
  22: CrashForExceptionInNonABICompliantCodeRange
  23: CrashForExceptionInNonABICompliantCodeRange
  24: CrashForExceptionInNonABICompliantCodeRange
  25: CrashForExceptionInNonABICompliantCodeRange
  26: CrashForExceptionInNonABICompliantCodeRange
  27: CrashForExceptionInNonABICompliantCodeRange
  28: CrashForExceptionInNonABICompliantCodeRange
  29: CrashForExceptionInNonABICompliantCodeRange
  30: CrashForExceptionInNonABICompliantCodeRange
  31: CrashForExceptionInNonABICompliantCodeRange
  32: <unknown>
  33: <unknown>
  34: <unknown>
  35: CrashForExceptionInNonABICompliantCodeRange
  36: <unknown>
  37: <unknown>
  38: <unknown>
  39: <unknown>
  40: <unknown>
  41: <unknown>
  43: <unknown>
  44: <unknown>
  45: <unknown>
  46: <unknown>
  47: <unknown>
  48: <unknown>
  49: <unknown>
  50: <unknown>
  51: <unknown>
  52: <unknown>
  53: <unknown>
  54: <unknown>
  55: <unknown>
  56: <unknown>
  57: <unknown>
  58: <unknown>
  59: <unknown>
  60: CrashForExceptionInNonABICompliantCodeRange
  61: CrashForExceptionInNonABICompliantCodeRange
  62: CrashForExceptionInNonABICompliantCodeRange
  63: <unknown>
  64: CrashForExceptionInNonABICompliantCodeRange
  65: BaseThreadInitThunk
  66: RtlUserThreadStart
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by bartlomieju** [comment]
Already fixed in #3728, pending release
